demo/tests/test_app.py

```diff
@@ -150,4 +151,44 @@ def test_multiple_input(self):
             for i in range(len(rows)):
                 val = rows.iloc[i][col]
                 true = (val - minmax[0]) / (minmax[1] - minmax[0])
-                self.assertAlmostEqual(enc.iloc[i][col], true, delta=constants.SLIDER_PRECISION)
\ No newline at end of file
+                self.assertAlmostEqual(enc.iloc[i][col], true, delta=constants.SLIDER_PRECISION)
+
+
+class TestPrescriptor(unittest.TestCase):
```

Comment:
Tests if the prescriptor files can load, if the prescriptions are coming out in the right shape, and if the prescriptions are scaled to the proper amount